packages/mdc-textfield/foundation.js

```diff
@@ -15,9 +15,11 @@
  * limitations under the License.
  */
 
+/* eslint-disable no-unused-vars */
 import MDCFoundation from '@material/base/foundation';
 import {MDCTextFieldAdapter, NativeInputType} from './adapter';
 import MDCTextFieldBottomLineFoundation from './bottom-line/foundation';
+import MDCTextFieldHelperTextFoundation from './helper-text/foundation';
```

Comment:
See above. Adding this line above line 22 ensures that the `no-unused-vars` check is only disabled for this one instance and not for the entire file.
`// eslint-disable-next-line no-unused-vars`